src/services/Config.ts

```diff
@@ -51,5 +53,10 @@ export default class Config {
     this.hostCurrency = 'XRP'
     this.hostAssetScale = 6
     this.hostCostPerMonth = Number(env.COST_PER_MONTH) || 10
+
+    // Admin API Config
+    this.adminApi = env.CODIUS_ADMIN_API === 'true'
```

Comment:
should we enable this by default? It might make more sense to have it off by default unless turned on...

Reply:
It's not enabled by default - the evaluation is handled in the same way as `CODIUS_HYPER_NOOP`, in that not specifying the variable will cause the evaluation to fail and return false since it will be `undefined`.